ui/apps/notifications/src/components/pages/customize-notification-page.tsx

```diff
@@ -293,11 +293,11 @@ const CustomizeNotificationPage: React.FC<CustomizeNotificationPageProps> = ({
         padding={'py-2,px-0'}
         customStyle="h-full md:h-min space-y-4"
       >
-        <Text variant="h5" align="center">
+        <Text variant="h5" align="center" customStyle="pb-2">
           {initial ? t('Customize Your Notifications') : t('Notifications Settings')}
         </Text>
-        <Divider customStyle="my-2" />
-        <Stack direction="column" customStyle="mx-4">
+        <Divider customStyle="!mt-0" />
```

Comment:
The `Divider` component has `apply` function used to overwrite styles. Did you encounter a situation where simply using `mt-0` isn't working for you ? Btw this comment applies for all cases you where you are using `! ` to override styles.

Reply:
Haha the approval came in as I was typing this comment :)

Reply:
It's a valid question, I assumed the styles weren't applying correctly, though it is preferable to not use `!`

Reply:
Ah I see Didd :)). Yes in this case simply using mt-0 didn't work to help remove the margin.

Reply:
I agree. And I was also curious to understand the situation where simply using the twind class isn't working ?